WordPress/Tests/Theme/SearchFormUnitTest.inc

```diff
@@ -0,0 +1,3 @@
+<?php
+// bad
+include( get_template_directory() . '/searchform.php' );
```

Comment:
Would `include( get_template_directory() . "/searchform.php" );` and `// searchform.php` be caught too?